### DISCUSSION TITLE
How to use bidirectional LSTM

### DISCUSSION THREAD

**Discussion by verina01** [timeseriesAI/tsai#645] [Q&A]
Hi, 

Thank you for this wonderful package. 
I'm new to programming and fastai. Because of my project, I had to jump into time series. 

I want to use Bidirectional LSTM for time series forecasting using this package. I have tried and got this error:

`len = 3`
`models = LSTMPlus(c_in=n_input, c_out=n_output, seq_len=len, bidirectional = True)`

`batch_tfms = TSStandardize()`
`model = TSForecaster(X, y, splits=splits, batch_tfms=batch_tfms, bs=512, arch=models, metrics=[mse, rmse])`
`TypeError : 'LSTMPlus' object has no attribute '__name__'`

Can you help me to use Bidirectional LSTM for time series forecasting correctly and predict 'targets' with this model ?

**Comment by oguiza** [comment]
Sorry for my late reply, @verina01. 
This issue is fixed now.
